packages/prisma/zod-utils.ts

```diff
@@ -374,7 +374,7 @@ export const bookingCancelSchema = z.object({
     .object({
       id: z.number(),
       name: z.string(),
-      cancellationReason: z.string().optional(),
+      cancellationReason: z.string().optional().nullable(),
```

Comment:
Let's do nullish here instead?
```suggestion
      cancellationReason: z.string().nullish(),
```